### DISCUSSION TITLE
202

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#183] [General]
# 潮流周刊第202期 - 夕阳西下

封面图拍摄于周末下午天气不错，4 点多就夕阳快下山，在杭州的小河边戴着 Bose QC45 骑着车，不冷还有点阳光的热量，拍到了远处建筑的这张照片，非常舒服惬意。

https://weekly.tw93.fun/posts/202-%E5%A4%95%E9%98%B3%E8%A5%BF%E4%B8%8B

**Comment by warden2018** [comment]
您的TESLA股票是怎么购买的？

**Reply by tw93** [reply]
好多年前弄过香港卡，富途长桥都可以

**Comment by FtdijsJin** [comment]
虽然但是，骑车带着耳机的话真的还挺危险的（关注点跑偏

**Reply by tw93** [reply]
有道理，我一般会把降噪模式关掉，找没有人的地方
